sdk/spring/spring-cloud-azure-starter-monitor/README.md

```diff
@@ -39,6 +39,12 @@ by [following the OpenTelemetry documentation](https://opentelemetry.io/docs/ins
 
 [Instruction][azure_native] for Spring Boot native image applications.
 
+### Enable OpenTelemetry features
+
+All the features described in the [OpenTelemetry Spring Boot starter][otel_spring_starter] documentation work with _Azure Monitor OpenTelemetry Distro / Application Insights in Spring Boot native image Java application_.
+
+_You have just to replace the `opentelemetry-spring-boot-starter` dependency by the `spring-cloud-azure-starter-monitor` one._
+
```

Comment:
I'm not personally a fan of lots of italics 😄, I think this would read fine (better to me) without any italics.

Reply:
I have removed them 😄